src/BloomExe/web/ServerBase.cs

```diff
@@ -8,6 +8,7 @@
 using System.Net;
 using System.Threading;
 using System.Windows.Forms;
+using Amazon.CloudSearch.Model.Internal.MarshallTransformations;
```

Comment:
I didn't add it...presumably Resharper did. No idea why. I'll see if it works without it.